addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -43,6 +43,7 @@ insertModelFields('mail.channel', {
             return this.currentPartnerId;
         },
     },
+    authorizedGroupFullName: { default: 'User types / Internal User' },
```

Comment:
Not correct, this was changed in python so that `group_public_id` is only set if `channel_type === channel`

Reply:
It's no longer a default after rebase.